pkg/operator/ceph/controller/controller_utils_test.go

```diff
@@ -133,6 +133,34 @@ func TestSetEnforceHostNetwork(t *testing.T) {
 	assert.False(t, EnforceHostNetwork())
 }
 
+func TestSetRevisionHistoryLimit(t *testing.T) {
+	logger.Infof("testing default value for %v", revisionHistoryLimitSettingName)
```

Comment:
Rook tries to avoid a ton of logging in tests. If we want to have each test be visually separated in the output, we can use `t.Run("test name", ...etc.. )` to run subtests.

Reply:
very good suggestion, thanks! changed from logging to `t.Run`